coins/verge.js

```diff
@@ -7,17 +7,17 @@
 
 module.exports = (callback, request) => {
     request({
-        uri: 'https://verge-blockchain.info/ext/getmoneysupply',
+        uri: 'https://verge-blockchain.info/api/moneysupply',
         rejectUnauthorized: false,
         json: false
```

Comment:
The new endpoint returns data in JSON format so this must be set to true.

Reply:
Whoops, fixed it!